pandas/core/arrays/base.py

```diff
@@ -687,7 +687,7 @@ def shift(self, periods: int = 1, fill_value: object = None) -> "ExtensionArray"
             a = empty
             b = self[:-periods]
         else:
-            a = self[abs(periods) :]
+            a = self[abs(periods):]
```

Comment:
does black not complain about these?  im pretty sure it introduced them

Reply:
Might be, apparently my black was on an older version. Will test tonight.